### ISSUE TITLE
No code output is shown in notebook

### ISSUE THREAD

**Issue by rwslippey** [jupyterhub/the-littlest-jupyterhub#644]
I've installed tljh on ubuntu 18.04. Although I do have a bit of a nonstandard setup, I'll explain more in a bit.

Executing anything in a notebook doesn't return any output. Even just a print("test") won't print any output.

I'm kind of new to notebooks, but my understanding is the * in the line means it's still executing? It never shows a number as I've seen before with a local notebook running on say conda. 

I did downgrade tornado as I've seen suggested elsewhere, which seemed to fix the connection to the kernel issues I was having but still no code output.

My non-standard config is just a firewall blocking everything from getting into the server (Digital Ocean Firewall) and I'm running everything over ZeroTier. I don't think this should be causing issues but I'm gonna open the firewall up here and test that theory.  EDIT: Just checked.... even direct over HTTP there is no change, still no output.

As for resources, I'm running 4GB on Digital Ocean so I don't see think is an issue there.

**Comment by rwslippey** [comment]
Okay, 

I must have messed something up in my configuration the first time around.
Reinstalled from scratch on new droplet. 

After getting through #642 and updating config things seems to be working as expected now.

`sudo tljh-config set limits.memory 4G
sudo tljh-config set limits.cpu 2
sudo tljh-config reload
`

**Comment by rwslippey** [comment]
Okay, just some more details here:

I had this working. Then installed the conda packages per [https://tljh.jupyter.org/en/latest/install/digitalocean.html#step-2-adding-more-users](url) and ran into the same issue. 

I decided to fall back to where I was originally by starting over again and after 3 or 4 reinstalls I wound up doing the following.

`sudo -E pip uninstall tornado` Based off another suggestion and errors in logs concerning tornado
`sudo -E pip install tornado==5.1.1`

No, luck. So I just upgraded tornado again.

`sudo -E pip install tornado --upgrade`
that seemed to do the trick and I followed the above to set config parameters. Unless I got a newer release (I didn't check prior to downgrading) it may have had an installation issue? For reference, I've got version 6.1 now and it's working so far.

Hope this helps someone. Happy holidays everyone!
